src/app/components/ButtonDefault/ButtonDefault.tsx

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
```

Comment:
Your button components look very similar. Maybe you could work with variants in your styling here? 
https://styled-system.com/variants/